tensorflow/contrib/learn/python/learn/ops/conv_ops.py

```diff
@@ -45,6 +45,7 @@ def conv2d(tensor_in,
     strides: A list of ints, 1-D of length 4. The stride of the sliding
       window for each dimension of input.
     padding: A string: 'SAME' or 'VALID'. The type of padding algorthim to use.
+      See the [comment here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn.py)
```

Comment:
https://www.tensorflow.org/versions/r0.8/api_docs/python/nn.html#convolution would that be a better link?  it links directly to the discussion of convolution...